### DISCUSSION TITLE
Qt package两个问题

### DISCUSSION THREAD

**Discussion by Elite-stay** [xmake-io/xmake#5559] [Q&A] [answered]
一个是package如果依赖Qt的内容了，能不能在package里add_frameworks，我试了在package描述域里写add_frameworks然后在工程全局描述域里add_packages是没效果
另一个是我需要在package里给target添加Qt，我在package的on_fetch里find_qt然后data_set似乎没效果，find_qt返回not nil了

**Comment by waruqi** [comment, marked as answer]
用 xmake-repo 里面的 qt 包

**Reply by Elite-stay** [reply]
libname类型不是table有点蛋疼，也行吧
